package.json

```diff
@@ -52,11 +54,12 @@
     "eslint-plugin-react": "^7.11.1",
     "eslint-plugin-redux-saga": "^0.10.0",
     "faker": "^4.1.0",
-    "file-loader": "^0.9.0",
+    "file-loader": "^4.1.0",
     "glob": "^6.0.4",
     "html-webpack-plugin": "^3.2.0",
     "jed": "^1.1.1",
     "jsel": "^1.1.6",
+    "mini-css-extract-plugin": "^0.8.0",
```

Comment:
What do we need this for? I did start using PF4 in cockpit but i didn't need this.